src/database_async.cc

```diff
@@ -79,22 +74,19 @@ void CloseWorker::WorkComplete () {
 
 /** IO WORKER (abstract) **/
 
-IOWorker::IOWorker (
-    Database *database
-  , Nan::Callback *callback
-  , leveldb::Slice key
-  , v8::Local<v8::Object> &keyHandle
-) : AsyncWorker(database, callback)
-  , key(key)
+IOWorker::IOWorker(Database *database,
+                   Nan::Callback *callback,
+                   const char *resource_name,
+                   leveldb::Slice key,
+                   v8::Local<v8::Object> &keyHandle)
+  : AsyncWorker(database, callback, resource_name), key(key)
```

Comment:
`IOWorker` is never used directly but instead inherited from classes seen below.